dist/rpm/os-autoinst.spec

```diff
@@ -25,8 +25,8 @@ Group:          Development/Tools/Other
 Url:            https://github.com/os-autoinst/os-autoinst
 Source0:        %{name}-%{version}.tar.xz
 %{perl_requires}
-%if 0%{?suse_version} > 1500
-# openSUSE Tumbleweed
+%if 0%{?suse_version} > 1500 || ( 0%{?is_opensuse} && 0%{?sle_version} >= 150400 )
```

Comment:
```suggestion
%if 0%{?suse_version} > 1500 || 0%{?sle_version} >= 150400
```

Reply:
OpenCV 4 is coming from SLE, so no need to have this openSUSE specific

Reply:
right , with sum we practically don't need this full condition